src/json-rpc-handlers/subscription/close.ts

```diff
@@ -25,8 +25,18 @@ export const handleSubscriptionsClose: JsonRpcHandler = async (
   context,
 ) => {
   const requestId = dwnRequest.id ?? uuidv4();
```

Comment:
Nit: can we rename `dwnRequest` to `jsonRpcRequest`?

Reply:
Done